cypress/component/Forms/SelectOptions.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import {mount} from 'cypress/react'
+import {FormField, FormFieldTypes} from '../../../src/components/forms/forms';
+import {asIdAndDisplayText, getFormattedName} from '../../../src/components/forms/SelectOptionInterface';
+import {StudyType} from '../../../src/components/forms/StudyType';
+import {SecondaryDataUseTerms} from '../../../src/components/forms/SecondaryDataUseTerms';
+import {NIHInstitutesAndCenters} from '../../../src/components/forms/NIHInstitutesAndCenters';
+import {DataTypes} from '../../../src/components/forms/DataTypes';
+
+describe('SelectOptions tests', () => {
+    describe('Loads a list of study options', () => {
+        it('should load a list of Study Type with options populated', () => {
+            const props = {
+                onChange: () => {
+                },
+                id: 'studyType',
+                title: 'Study Type',
+                type: FormFieldTypes.SELECT,
+                selectOptions: asIdAndDisplayText(StudyType.studyTypeList),
+                isMulti: true
+            };
+            cy.spy(props, 'onChange');
+
+            mount(<FormField {...props}/>);
+            cy.get('#studyType').type('Obs{enter}');
+            cy.get('#studyType').then(() => {
+                expect(props.onChange).to.be.calledWith({
+                    key: 'studyType',
+                    value: [{key: StudyType.OBS.key, displayText: StudyType.OBS.name}],
+                    isValid: true
+                });
+            });
+        });
+        it('should load a list of Secondary Data Use Terms with options populated', () => {
+            const props = {
+                onChange: () => {
+                },
+                id: 'secondaryDataUse',
+                title: 'Secondary Data Use',
+                type: FormFieldTypes.SELECT,
+                selectOptions: asIdAndDisplayText(SecondaryDataUseTerms.secondaryDataUseTermsList),
```

Comment:
This is the kind of dependency I think it would be better to avoid. Unless this is testing the behavior of `SecondaryDataUseTerms.secondaryDataUseTermsList`, I think it would be better to use test-only data to validate the behavior of `FormField`. That way this test doesn't need to change if the data happens to change.